From 902fff972891be0938c60799d0c9789cd53f571c Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Mon, 22 Aug 2022 15:01:28 +0300
Subject: [PATCH] testlib: don't track the build or source directories

Build tests without location of the build and sources directories.

Upstream-Status: Inappropriate [embedded specific]
Change-Id: I8d5add473623a3d9f481097649819c9fb906e4b2
---
 mkspecs/features/testlib_defines.prf     | 2 --
 src/testlib/Qt5TestConfigExtras.cmake.in | 5 -----
 2 files changed, 7 deletions(-)

diff --git a/mkspecs/features/testlib_defines.prf b/mkspecs/features/testlib_defines.prf
index 901e03a91d1..e69de29bb2d 100644
--- a/mkspecs/features/testlib_defines.prf
+++ b/mkspecs/features/testlib_defines.prf
@@ -1,2 +0,0 @@
-contains(TEMPLATE, vc.*): DEFINES += QT_TESTCASE_BUILDDIR=\"$$OUT_PWD\"
-else: DEFINES += QT_TESTCASE_BUILDDIR=$$shell_quote(\"$$OUT_PWD\")
diff --git a/src/testlib/Qt5TestConfigExtras.cmake.in b/src/testlib/Qt5TestConfigExtras.cmake.in
index 2a575958ae4..e69de29bb2d 100644
--- a/src/testlib/Qt5TestConfigExtras.cmake.in
+++ b/src/testlib/Qt5TestConfigExtras.cmake.in
@@ -1,5 +0,0 @@
-
-set_property(TARGET Qt5::Test
-  APPEND PROPERTY
-    INTERFACE_COMPILE_DEFINITIONS QT_TESTCASE_BUILDDIR=\\\"\${CMAKE_BINARY_DIR}\\\"
-)
